packages/lundium/package.json

```diff
@@ -1,9 +1,10 @@
 {
-	"name": "@lundium/components",
+	"name": "lundium",
 	"version": "0.0.1",
 	"license": "UNLICENSED",
 	"description": "Components",
-	"main": "./src/index.js",
+	"main": "./dist",
+	"sideEffects": false,
```

Comment:
are you sure?

Reply:
Since there will be only UI components in this library, there shouldn't be any side-effects right? 
But maybe I'm missing something

Reply:
there are... there will e.g. manipulation of the DOM